### ISSUE TITLE
Crossing Streams for Job Search button vs job in table

### ISSUE THREAD

**Issue by vortexing** [FredHutch/batch-dashboard#22]
When I go to the Jobs tab and put in a jobid like (35b31082-470c-426f-8325-7e431912f841), and click the "search" button, the window that pops up appears to show two different jobs as different attempts of this jobID but they are not.  Included in this are links to the second, independent jobs' logs, the lists of job inputs for both jobs are included all together as one list, etc.  

![Screen Shot 2020-06-09 at 9 30 40 AM](https://user-images.githubusercontent.com/13182518/84174958-1d035380-aa34-11ea-9e85-9e8ba8b7f483.png)

However, if I click on the jobid in the table, the window that pops up shows the job information correctly.  

![Screen Shot 2020-06-09 at 9 32 50 AM](https://user-images.githubusercontent.com/13182518/84175018-386e5e80-aa34-11ea-9718-dd6189994f8d.png)

**Comment by dtenenba** [comment]
Hmm....when I paste the job id into the box, I only see one attempt:

![image](https://user-images.githubusercontent.com/2286826/84179288-943be600-aa3a-11ea-9bc5-edcaf85251f7.png)

But then I tried it again and saw two attempts.

When I describe the job using the CLI, I see two attempts:

```
$ aws batch describe-jobs --jobs 35b31082-470c-426f-8325-7e431912f841 |grep '"container":'
                    "container": {
            "container": {
```

(The `container` key shows up once for each attempt.)

I ran that a whole bunch of times and it seemed to consistently return two attempts.

So....it does seem like there were two attempts for that job. If that is wrong, then the problem is at the AWS Batch level, not the dashboard level. But what happens and where you go when you clic on those View Logs links could definitely be a dashboard issue. I will look into this some more.

**Comment by dtenenba** [comment]
Yeah, this is a UI thing. If I close the dialog and click Search again, I then see another attempt, and I see an additional attempt each time I do that. They all seem to be the same (the giveaway is the identical start times and end times) and clicking on the View Logs brings me to the same page (which seems to be the correct page) whichever one I click on.

I am not understanding your description which seems to suggest that info from separate unrelated jobs is being shown in this dialog. To me it seems like info from this one (correct) job is being duplicated but I am not seeing any foreign unrelated job info sneaking in there. Similarly, clicking on View Logs seems to take me to the right place.

If I am wrong about this can you describe in more detail exactly where you are seeing separate jobs' info displayed in that dialog or in the View Logs page?

**Comment by vortexing** [comment]
Now that the jobs are all done, I can no longer reproduce them either.  It only happened the 3 times I reloaded the app when the jobs were still running.  If I see it again, I'll post more info.
